### PR TITLE
Apply rights naming convention

### PR DESCRIPTION
**Pull request by ildyria**
Counter part of https://github.com/LycheeOrg/Lychee/pull/1539

### CODE REVIEW

**Review by qwerty287 — APPROVED** [review]
Aside the comment about commented code (see above), this looks good to me.

File: scripts/main/albums.js [hunk 1]
```diff
@@ -128,8 +129,8 @@ albums.localizeSmartAlbums = function (data) {
 	if (data.public) {
 		data.public.title = lychee.locale["PUBLIC"];
 		// TODO: Why do we need to set these two attributes? What component relies upon them, what happens if we don't set them? Is it legacy?
-		data.public.is_public = true;
-		data.public.requires_link = true;
+		// data.public.is_public = true;
+		// data.public.is_link_required = true;
```
[the remark this answers is not in the document]
Comment (ildyria) [reply scripts/main/albums.js]: removed completely.

**Comment by ildyria** [comment]
> Found another issue: updating login credentials doesn't work, you changed the API endpoint but not the queried API in the UI. Need to change `Settings::updateLogin` to `User::updateLogin`.

Fixed: [e588007](https://github.com/LycheeOrg/Lychee-front/pull/320/commits/e588007b1dcc7fac430f21b7e56e77c103fd0eb2)

**Review by qwerty287 — COMMENTED** [review]
Let me sum up the issue again:
1. You changed only one API to `updateLogin`. Also change it in line 217. Are you sure that the change you did is correct?
2. If you have upload privileges, the upload button is shown on all albums, but you get an error (of course) if you're not the owner.
3. The photo visibility dialog enables the `Public` switch if you never selected something from it and saved it. The photo is not public though.

**Comment by ildyria** [comment]
> Let me sum up the issue again:
> 
> 1. You changed only one API to `updateLogin`. Also change it in line 217. Are you sure that the change you did is correct?
> 2. If you have upload privileges, the upload button is shown on all albums, but you get an error (of course) if you're not the owner.
> 3. The photo visibility dialog enables the `Public` switch if you never selected something from it and saved it. The photo is not public though.

1. should be fixed with [93f5fe7](https://github.com/LycheeOrg/Lychee-front/pull/320/commits/93f5fe759b7a6c74a6df033fe6d1fc96e9776e33)
2. should be fixed with [fbb8ade](https://github.com/LycheeOrg/Lychee/pull/1539/commits/fbb8adedab5800acdcc8267a8beb734ae23f9e20)
3. should be fixed with [93f5fe7](https://github.com/LycheeOrg/Lychee-front/pull/320/commits/93f5fe759b7a6c74a6df033fe6d1fc96e9776e33).

**Review by qwerty287 — APPROVED** [review]

**Pull request merged**